crates/subspace-node/Cargo.toml

```diff
@@ -88,7 +88,8 @@ thiserror = "2.0.0"
 tokio = { version = "1.40.0", features = ["macros"] }
 tokio-stream = "0.1.16"
 tracing = "0.1.40"
-tracing-subscriber = { version = "0.3.18", features = ["env-filter"] }
+tracing-subscriber = { version = "0.3.18", features = ["env-filter", "json"] }
```

Comment:
This comment means we can delete this line entirely:
> The dependency only needs to be added to crates where it is used and remove tracing-subscriber from direct dependencies where you use subspace-logging since it will no longer be necessary.

Reply:
Oh ok :) see it.